Sources/Helpers/OANativeUtilities.h

```diff
@@ -52,4 +52,6 @@
 
 + (QList<OsmAnd::TileId>)convertToQListTileIds:(NSArray<NSArray<NSNumber *> *> *)tileIds;
 
++ (CLLocation *)parseLatLon:(NSString *)latLon;
```

Comment:
This method should not be in the NativeUtilities but in OAUtilities (native utilities is a class for the c++ interaction)